### PR TITLE
Fixes bitrunner glitches not spawning if the reference list has a invalid ref

### PR DESCRIPTION
**Pull request by Arturlang**
…has a null ref, somehow

## About The Pull Request
See title

## Why It's Good For The Game
Most of the time, due to this, a glitch won't even spawn

## Proof Of Testing
Went ingame, tested it, spawned as a glitch

## Changelog

:cl:
fix: Bitrunner glitches should now spawn properly more often
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
Going to test if this issue is present on /TG/

**Comment by Arturlang** [comment]
Can't reproduce on TG as they are missing a bitrunner map

**Review by Majkl-J — COMMENTED** [review]
Is this bound to a specific map?

**Comment by Arturlang** [comment]
> Is this bound to a specific map?

Issue was found on a specific map, but its a good fix honestly

**Comment by Majkl-J** [comment]
> Issue was found on a specific map, but its a good fix honestly, as for the actual reason its caused uhhh, probably some mapping issue, though im wondering why it even attempts to iterate through a null object and how that even gets added

Yea I'd rather look for the source than to obfuscate it via nullcheck in this case, if you tell me which map it is I'll look into it with debug tools

File: code/modules/bitrunning/server/threats.dm [hunk 1]
```diff
@@ -155,7 +155,7 @@
 /// Removes any invalid candidates from the list
 /obj/machinery/quantum_server/proc/validate_mutation_candidates()
 	for(var/datum/weakref/creature_ref as anything in mutation_candidate_refs)
-		var/mob/living/creature = creature_ref.resolve()
+		var/mob/living/creature = creature_ref?.resolve()
```
Comment (nevimer) [inline code/modules/bitrunning/server/threats.dm:158]:
```suggestion
		var/mob/living/creature = creature_ref?.resolve() // BUBBER EDIT - REF CHECK for RUNTIME AVOIDANCE
```

**Review by nevimer — APPROVED** [review]

**Pull request merged**